src/components/EntityPage/Participant/Utils/sanitize.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
❤️ 
Nice function!

Just be extra careful not to mutate the parameter: you might have a nasty surprise if you pass the value to another function that expect the value _not_ to be modified.

Those functions should return new values. Either ensure the immutability of the function, or clone it's input before mutating it. Since that one is recursive, cloning you have a bigger impact, so I would suggest just to keep immutability.

If I'm not clear (sorry, in a hurry), I'll show you tomorrow what I mean.

Reply:
Here's an example of a way to achieve this with `Object.prototype.keys` and `Map.prototype.reduce`:

```js
function sanitize(obj) {
  if (Array.isArray(obj)) return obj.map(sanitize);

  return Object.keys(obj).reduce((newObj, key) => {
    const val = obj[key];
    newObj[key] = val === null ? '--' : typeof val === 'boolean' ? `${val}` : obj[key];
    return newObj;
  }, {});
}
```

I haven't tested, though.

Reply:
Alright! I tested it and I works ^^

I didn't know that JS had reduce, I'll keep that in mind for the future :) Thanks!